wagtail/contrib/modeladmin/tests/test_simple_modeladmin.py

```diff
@@ -263,6 +263,43 @@ def test_col_extra_attributes(self):
         """
         self.assertContains(response, test_html, html=True)
 
+    def test_linked_name_column(self):
+        response = self.get()
+        self.assertEqual(response.status_code, 200)
+        # The InspectView is enabled in the AuthorModelAdmin, so the inspect-button is
+        # used for the link.
+        test_html = """
+            <div class="title-wrapper"><a href="/admin/modeladmintest/author/inspect/1/" title="Inspect this author">J. R. R. Tolkien</a></div>
+        """
+        self.assertContains(response, test_html, html=True)
+
+    @mock.patch(
+        "wagtail.contrib.modeladmin.helpers.permission.PermissionHelper.user_can_inspect_obj",
+        return_value=False,
+    )
+    def test_linked_name_column_without_inspect_view(self, *mocks):
+        response = self.get()
+        self.assertEqual(response.status_code, 200)
+        # With a disabled InspectView, the edit button is the primary action.
+        test_html = """
+            <div class="title-wrapper"><a href="/admin/modeladmintest/author/edit/1/" title="Edit this author">J. R. R. Tolkien</a></div>
+        """
+        self.assertContains(response, test_html, html=True)
+
+    @mock.patch(
+        "wagtail.contrib.modeladmin.helpers.permission.PermissionHelper.user_can_inspect_obj",
+        return_value=False,
+    )
+    @mock.patch(
+        "wagtail.contrib.modeladmin.helpers.permission.PermissionHelper.user_can_edit_obj",
+        return_value=False,
+    )
+    def test_no_linked_name_column_with_missing_permissions(self, *mocks):
```

Comment:
```suggestion
    def test_title_column_is_not_linked_when_user_has_cannot_edit_or_inspect(self, *mocks):
```

Reply:
I will skip "_has", when I change the tests.

Reply:
Oops :)

Reply:
amended the above suggestion :P

Reply:
@zerolab that's not patronising at all :)

Reply:
The road to hell is paved with good intentions. I am sorry @ababic